### DISCUSSION TITLE
Diagrama Modelamiento de Negocio

### DISCUSSION THREAD

**Discussion by ChanLuis** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#160] [Q&A]
![Modelamiento de Negocio](https://user-images.githubusercontent.com/99062113/155907793-92ef14b6-5088-4227-86fc-35a5b4842fad.png)

Hola amigos, será que alguien me puede decir si voy bien. Estoy algo atrasado.

**Comment by doguedogue** [comment]
Se ve bien, sigue así 👍🏼

**Comment by adrezpe** [comment]
Me gusta muchisimo el flujo, vas muy bien

**Comment by DanyVeneno** [comment]
Tu dale Chan
